rest-service/manager_rest/test/endpoints/test_filters.py

```diff
@@ -0,0 +1,36 @@
+from manager_rest.test import base_test
+from manager_rest.storage.models_base import db
+from manager_rest.storage.filters import add_labels_filters_to_query
+from manager_rest.storage.resource_models import Deployment, DeploymentLabel
+
+
+class FiltersFunctionalityTest(base_test.BaseServerTestCase):
+
+    LABELS = [{'a': 'b'}, {'a': 'z'}, {'c': 'd'}]
+    LABELS_2 = [{'a': 'b'}, {'c': 'z'}, {'e': 'f'}]
+
+    def test_filters_functionality(self):
```

Comment:
Is it worth adding a test to ensure that we complain if passed an invalid label in a filter expression?

Reply:
Of course, my bad, adding.